src/components/organisms/Table/Table.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@utils/testing-utils';
+import React from 'react';
+import { Table } from './Table';
+
+describe('Table', () => {
```

Comment:
We may need to include the tests for store dispatching in the future, otherwise coverage won't be met

Reply:
Coverage is already met for this component but I agree that it is weird, maybe the way coverage is calculated right now is wrong.
I also agree that tests should be added for store dispatching, I was thinking adding them when including the CardPile in the Table

Reply:
I am ok to do it on the next PR, we will have more use cases for it in this case